hack/sign-and-notarize-cli.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+echo "checking required environment variables are set..."
+
+if [[ -z "${KARGO_BIN_PATH}" ]]; then
+    echo "KARGO_BIN_PATH is unset." && exit 1
+fi
+
+if [[ -z "${QUILL_SIGN_P12}" ]]; then
+    echo "QUILL_SIGN_P12 is unset." && exit 1
+fi
+
+if [[ -z "${QUILL_SIGN_PASSWORD}" ]]; then
+    echo "QUILL_SIGN_PASSWORD is unset." && exit 1
+fi
+
+if [[ -z "${QUILL_NOTARY_KEY}" ]]; then
+    echo "QUILL_NOTARY_KEY is unset." && exit 1
+fi
+
+if [[ -z "${QUILL_NOTARY_KEY_ID}" ]]; then
+    echo "QUILL_NOTARY_KEY_ID is unset." && exit 1
+fi
+
+if [[ -z "${QUILL_NOTARY_ISSUER}" ]]; then
+    echo "QUILL_NOTARY_ISSUER is unset." && exit 1
+fi
+
+echo "confirmed environment variables are set correctly."
+
+if [[ ! -x "$(command -v quill)" ]]; then
+    echo "quill is not installed. Installing..."
+    curl -sSfL https://get.anchore.io/quill | sudo sh -s -- -b /usr/local/bin \
```

Comment:
Can you ensure this is stored in `./hack/bin`, and that it is pinned to a version?